### ISSUE TITLE
Cannot execute drush uli with [username|UID] as parameter

### ISSUE THREAD

**Issue by klonos** [backdrop-contrib/drush#227]
These work:

`drush uli` (default to UID 1)
`drush uli --uid=2`
`drush uli --name=user`
`drush uli --mail=some@email.com`

These do not work in Backdrop (no error - no output), whereas they work in Drupal:

`drush uli 2`
`drush uli user`

I am not sure if they ever worked in Backdrop, but if we regressed at some point, then most likely would be with #156

**Comment by jackaponte** [comment]
> I am not sure if they ever worked in Backdrop, but if we regressed at some point, then most likely would be with #156

AFAIK those have never worked!

**Comment by serundeputy** [comment]
Yep. they never worked; I was not aware of them. We can make them work.

**Comment by ghost** [comment]
I just noticed this today and was going to file an issue. The code for using the provided username is simply missing from the command. There are checks to see if it's empty, or if the `--` options are provided (which take precedence), but none if it actually *is* provided...
